app/src/lib/lookups.js

```diff
@@ -1,4 +1,7 @@
-import { TOKEN_TYPE, concept_with_sense, split_stem_and_sense } from './parser/token'
+import { TOKEN_TYPE, split_stem_and_sense } from './parser/token'
```

Comment:
these imports really tell a cohesive story about what is going to happen in this file, great sign!